integration-tests/environments/react-native-test-app/react-native.config.js

```diff
@@ -16,17 +16,26 @@
 //
 ////////////////////////////////////////////////////////////////////////////
 
-const path = require("node:path");
-const { getDefaultConfig, mergeConfig } = require("@react-native/metro-config");
+const project = (() => {
+  try {
+    const { configureProjects } = require("react-native-test-app");
+    return configureProjects({
+      android: {
+        sourceDir: "android",
+      },
+      ios: {
+        sourceDir: "ios",
+      },
+      windows: {
```

Comment:
Preparing for RN Windows support? 🆒

Reply:
I didn't check windows when generating the app, but this got included anyway. @tido64 do you know why that happened?

Reply:
IIRC, it is always included. It won't affect anything.